src/Standards/BestIt/CodeSniffer/File.php

```diff
@@ -12,78 +13,14 @@
  * Class File
  *
  * Wrapper Class for PhpCsFile to provide a consistent way to replace int|bool returns
- * with int returns (false => -1)
+ * with int|bool returns false
  * Additionally there could be some architecture changes in the future, like Token-Objects and so on.
  *
  * @author Nick Lubisch <nick.lubisch@bestit-online.de>
  * @package BestIt\CodeSniffer
  */
 class File extends AbstractFileDecorator
```

Comment:
Is there any real value of this class after your changes? If not, remove it!

Reply:
There are still methods defined in this class. I could remove the position method and another one that now would be identical to standard.